src/main/java/orm/ColumnMeta.java

```diff
@@ -0,0 +1,18 @@
+package orm;
+
+import jakarta.persistence.Column;
+
+public record ColumnMeta(
+        boolean nullable,
+        int length
+) {
+    public static ColumnMeta from(Column column) {
+        var nullable = column == null ? true : column.nullable();
```

Comment:
```suggestion
        var nullable = column == null || column.nullable();
```
간단하게 표현하는게 더 좋을 수 도 있겠네요 😄

Reply:
넵 이부분도 수정하도록 하겠습니다.